webpack.config.js

```diff
@@ -22,6 +22,8 @@ module.exports = {
     "tco-calculator": "./static/js/src/tco-calculator.js",
     "sticky-nav": "./static/js/src/sticky-nav.js",
     imageBuilder: "./static/js/src/imageBuilder.js",
+    lightbox: "./static/js/src/lightbox.js",
+    appliance: "./static/js/src/appliance.js"
```

Comment:
The JS is not added to the page at all.

Reply:
ugh, I somehow blatted my changes with the rebase. Hold on